content/en/getting_started/agent/_index.md

```diff
@@ -12,90 +12,219 @@ further_reading:
+### About the Agent
+
+The Datadog Agent is software that runs on your hosts. It collects events and metrics from hosts and sends them to Datadog, where you can analyze your monitoring and performance data. It can run on your local hosts (Windows, MacOS), containerized environments (Docker, Kubernetes), on premises data centers, and supports configuration management tools (Chef, Puppet, Ansible). The Agent is able to collect 75-100 system level metrics every 15-20 seconds. With additional setup and configuration, the Agent can also report live processes, logs, and traces to the Datadog Platform. The Datadog Agent is open source and its source code is available on GitHub at [DataDog/datadog-agent][1].
+
+### Agent overhead
+
+The amount of space and resources the Agent takes up depends on the configuration and what data the Agent pulls. At the onset, you can expect around 0.08% CPU used on average with a disk spalce of roughly 830MB to 880MB. 
+
+See the [Agent Overhead][2] to learn more about these benchmarks.
+
+### Data Collected
+
+#### Agent metrics
+
+| Metric                           | Description                                                                                                          |
+| -------------------------------- | -------------------------------------------------------------------------------------------------------------------- |
+| **datadog.agent.python.version** | Shows a value of `1` if the Agent is reporting to Datadog. The metric is tagged with the `python_version`. |
+| **datadog.agent.running**        | Shows a value of `1` if the Agent is reporting to Datadog.                                                 |
+| **datadog.agent.started**        | A count sent with a value of `1` when the Agent starts (available in v6.12+).                                        |
+
+See the [Agent Metrics][3] integration for a full list of Agent metrics.
+
+#### Checks
+
+Depending on your platform, the Agent has several core checks enabled by default that collect metrics.
+
+| Check       | Metrics       | Platforms          |
+| ----------- | ------------- | ------------------ |
+| CPU         | [System][4]  | All                |
+| Disk        | [Disk][5]    | All                |
+| IO          | [System][4]  | All                |
+| Memory      | [System][4]  | All                |
+| Network     | [Network][6] | All                |
+| NTP         | [NTP][7]     | All                |
+| Uptime      | [System][4]  | All                |
+| File Handle | [System][4]  | All except Mac     |
+| Load        | [System][4]  | All except Windows |
+| Docker      | [Docker][8]  | Docker             |
+| Winproc     | [System][4]  | Windows            |
+
+To collect metrics from other technologies, see the [Integrations][9] page.
+
+## Containerized Environments
+
+There are a few differences in the way the Agent runs in a containerized environment.
```

Comment:
You haven't yet said how the Agent is configured on a host, so it's a little premature to compare how it's configured on a container. Maybe we should call this section "Differences between Agents for hosts and containers". Then this sentence would be "There are a few differences in the way the Agent runs in a containerized environment from how it runs on a host."

Reply:
Would this flow better if this section and the "Why Should I Install the Agent" section were moved below the walkthrough portion? So a user would see how to install and setup an Agent, and then they can see how this is different in a container environment.